src/native/src/core/texture_manager.cpp

```diff
@@ -53,7 +53,7 @@ void texture_manager::add_texture(mc_atlas_texture & new_texture, atlas_type typ
     GLenum format = GL_RGB;
     switch(new_texture.num_components) {
         case 1:
-            format = GL_R;
+            format = GL_RED;
```

Comment:
from here:
https://www.opengl.org/sdk/docs/man2/xhtml/glTexGen.xml

Specifies a texture coordinate. Must be one of GL_S, GL_T, GL_R, or GL_Q.

So GL_R had something to do with texture coords in old opengl versions.


As far as i can tell they never changed it. Its always been that way. The values of each are also different
GL_RED: 0x1903
GL_R: 0x2002